.github/workflows/ci-pull-request-status.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches:
+      - develop
+      - release/**
+  workflow_dispatch:
+    # empty as it is used only to manually trigger the workflow
+
+env:
+  NODE_VERSION: "16"
+  GH_TOKEN: ${{ github.token }}
```

Comment:
for this, I'm sure this is fine because it's required for the user to be able to make use of the gh cli right? 

but the previous one from the previous yml is not needed.

Reply:
GH_TOKEN is indeed required by GH CLI.  I removed NODE_VERSION